torchao/quantization/prototype/qat.py

```diff
@@ -47,6 +47,7 @@ def prepare(
             *args: Any,
             **kwargs: Any
         ) -> torch.nn.Module:
+            state_dict = model.state_dict()
```

Comment:
oh OK, right, this function have to do create_quantized_state_dict() and load_state_dict() in the gpt-fast API, but I feel we could also change `_replace_linear_8da4w` to instantiate from the existing floating point module, that might be clearer

Reply:
Yeah I agree, updated